### ISSUE TITLE
Black screen on Chrome/Safari iPhone (SE 10) 

### ISSUE THREAD

**Issue by timdonovanuk** [thecodingmachine/workadventure#2086]
Hi,

I see iPhone support was patched in a while back but it seems there has been some regression. I have tested connecting to the official server on an iPhone (SE 10) in Chrome and Safari and just instantly get a black screen. I can see it has loaded the favicon so it has at least made a connection but little else.

Thanks.

**Comment by moufmouf** [comment]
Yes, we are aware of this. This is due to an updated Phaser 3 package that is using advanced JS features not available before iOS 15.

We are currently working on a patch to display the minimum version supported for each browser: https://github.com/thecodingmachine/workadventure/pull/1995
